heroku_connect/test/utils.py

```diff
@@ -52,3 +57,25 @@ def heroku_cli(stdout='', stderr='', exit_code=0):
     os.chmod(exec_name, st.st_mode | stat.S_IEXEC)
     yield
     os.environ['PATH'] = path
+
+
+@contextmanager
+def no_heroku_connect_write_restrictions():
+    """
+    Allow writing to read-only Heroku Connect tables.
+
+    Can be used to create test data, e.g.::
+
+        def get_test_data():
+            with no_heroku_connect_write_restrictions():
+                return MyReadOnlyModel.objects.create()
+
+    """
+    from ..conf import settings
+    new_settings = copy.copy(getattr(settings, 'DATABASE_ROUTERS', []))
```

Comment:
Instead of this - how about -

    new_settings = []
    if getattr(settings, 'DATABASE_ROUTERS'):
        new_settings = [_ for _ in settings.DATABASE_ROUTERS if _ != 'heroku_connect.db.router.HerokuConnectRouter']

or even

    new_settings = getattr(settings, 'DATABASE_ROUTERS', []) and [_ for _ in settings.DATABASE_ROUTERS if _ != 'heroku_connect.db.router.HerokuConnectRouter']

Reply:
Ut doesn't need to be a list, it could be a tuple. (I would actually recommend a tuple.)
I don't want to change the type or assume it's always a list. It could be anything for that matter. As long as its implements `__copy__` and `remove`, it works :)